### DISCUSSION TITLE
여섯번째 모임 후기

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#17] [모임 후기!]
어제 모임에 대한 내용을 노션에 정리했었는데, 노션 기능을 통해서 요약 및 액션 만들기를 하고,조금 다듬어보았습니다! 쓸만하네요!ㅎㅎ

(사진)

내 인생에 진짜 도움이 되는 것이 뭘까?
=> 하기 전, 하고 있을 때보다 하고 나서 좋은 것!
=> 나를 넘어, 다른 사람에게까지 인정받고, 영향을 미치는 것!

## 모임 요약
- 현재까지 총 5번의 모임 진행, 새로운 사람들과의 네트워킹 및 성장 관련 생산적 대화에 긍정적 평가
- OKR 관리 개선 필요성 인식 => 시스템으로 보완하기!
- 목표: 8명의 꾸준한 참여자 확보
- 실험: "성장한 것을 확인시켜주자" - OKR 점검, 체계 구축, 공통 목표 설정
- LinkedIn 활용 및 초대 시스템을 통한 멤버 확대 실험 계획
- 커리어 발전을 위한 3가지 조언 공유 및 토론

### 개인 활동
- [ ]  OKR 관리 시스템 개선하기
- [ ]  매주 월요일 OKR 리뷰 및 실행 계획 공유하기
### 팀 활동
- [ ]  12월 12일까지 공동 목표 설정 및 달성 계획 수립하기
### 모임 활성화를 위한 실험
- [ ]  월 8명의 꾸준한 참여자 확보를 위한 전략 수립하기
- [ ]  활동 규칙 (시간 정렬, 카톡 공유, 결과 수치화) 구체화하기
- [ ]  새로운 멤버를 위한 온보딩 규칙 만들기
- [ ]  기존 멤버를 통한 '초대권' 시스템 구축하기
- [ ]  LinkedIn에 가짜연구소 후기 작성하기

**Comment by yunhee1** [comment]
지금까지의 스터디를 회고하면서 스터디의 정체성을 함께 만들어가는 기분이 들어 좋았습니다! 
만나지 않는 동안에도 개인의 OKR을 상기하고 꾸준히 이어나갈 수 있도록 하는 방법을 다음 만남때까지 좀 더 고민해서 가져가겠습니다!
